controllers/service.go

```diff
@@ -46,15 +46,15 @@ func (r *DirectoryServiceReconciler) reconcileService(ctx context.Context, ds *d
 func createService(ds *directoryv1alpha1.DirectoryService, svc *v1.Service) error {
 	svcTemplate := v1.Service{
 		ObjectMeta: metav1.ObjectMeta{
-			Labels:      make(map[string]string),
+			Labels:      createLabels(ds.Name, nil),
 			Annotations: make(map[string]string),
 			Name:        ds.Name,
 			Namespace:   ds.Namespace,
 		},
 		Spec: v1.ServiceSpec{
 			ClusterIP: "None", // headless service
 			Selector: map[string]string{
-				"app.kubernetes.io/name":     ApplicationName,
+				"app.kubernetes.io/name":     LabelApplicationName,
```

Comment:
why not use `createLabels` here?

Reply:
This is just for the selector - as long as it is unique it will select the right pods.

Reply:
That's right. However, I think it would be better to just keep all the labels in the same place. I don't think there's an advantage to spell those out here.